battleground-state-changes.html.tmpl

```diff
@@ -65,11 +75,6 @@
         This website is <a href="https://github.com/alex/nyt-2020-election-scraper">open source</a> and was written by <a href="https://github.com/alex/nyt-2020-election-scraper/graphs/contributors">these people</a>.
     </p>
 
```

Comment:
@NikilXYZ - did you intend to delete the following stuff around table shrinking?

Reply:
whoop - disregard my comment. Looks like it was just caught in a rebase.